lexicon/conversion.py

```diff
@@ -140,6 +140,66 @@ def run(self):
             lexicon_file.write(ET.tostring(root, "unicode"))
 
 
+class LexiconFromTextFileJob(Job):
+    """
+    Create a bliss lexicon from a regular text file, where each line contains:
+    <WORD> <PHONEME1> <PHONEME2> ...
+    separated by tabs or spaces.
+    The lemmata will be added in the order they appear in the text file,
+    the phonemes will be sorted alphabetically.
+    Phoneme variants of the same word need to appear next to each other.
+    No special lemmata or phonemes are added.
+
+    As the splitting is taken from RASR and not fully tested,
+    it might not work in all cases so do not use this job
+    without checking the output manually on new lexica.
```

Comment:
Please also add a warning that the generated lexicon is missing special lemmas and thus not directly usable in RASR.